test/unit/annotation_spec.js

```diff
@@ -1638,6 +1638,41 @@ describe("annotation", function () {
           done();
         }, done.fail);
     });
+
+    it("should render splited text for printing", function (done) {
+      textWidgetDict.set("Ff", AnnotationFieldFlag.MULTILINE);
+
+      const textWidgetRef = Ref.get(271, 0);
+      const xref = new XRefMock([
+        { ref: textWidgetRef, data: textWidgetDict },
+        fontRefObj,
+      ]);
+      const task = new WorkerTask("test print");
+      partialEvaluator.xref = xref;
+
+      AnnotationFactory.create(
+        xref,
+        textWidgetRef,
+        pdfManagerMock,
+        idFactoryMock
+      )
+        .then(annotation => {
+          const id = annotation.data.id;
+          const annotationStorage = {};
+          annotationStorage[id] = "a aa aaa aaaa aaaaa aaaaaa";
```

Comment:
Can we make this string so that it also covers the "break in the middle of string" (no whitespace found) codepath, since that seems to be uncovered here? Perhaps introducing a large word would already be enough. If this line becomes longer than 80 characters, then concatenate smaller substrings.